gatorgrade/main.py

```diff
@@ -2,6 +2,7 @@
 from pathlib import Path
 import typer
 
+
```

Comment:
This should probably not be here, it may cause merge conflicts. I don't think we needed to make any changes in main.

Reply:
no changes have been made to main.py just merging main into the branch